### PR TITLE
lets you lock direction in a mech

### PR DESCRIPTION
**Pull request by MadmanMartian**
what it says on the tin. You can lock direction in a mech, and when you go to move, your mech won't turn arduously in that direction.

:cl:
* rscadd: You can now lock direction in a mech

### CODE REVIEW

File: code/game/mecha/mecha.dm [hunk 1]
```diff
@@ -1263,6 +1266,16 @@
...
 
+/obj/mecha/verb/lock_direction()
+	set name = "Lock direction"
+	set category = "Exosuit Interface"
+	set src = usr.loc
+	set popup_menu = 0
+	if(usr != src.occupant)
+		return
+	lock_dir = !lock_dir
+	return
```
Comment (PJB3005) [inline code/game/mecha/mecha.dm]: It trails!

File: code/game/mecha/mecha.dm [hunk 2]
```diff
@@ -43,7 +43,7 @@
 	var/lights = 0
 	var/lights_power = 6
 	var/rad_protection = 50 	//How much the mech shields its pilot from radiation.
-
+	var/lock_dir = 0
```
Comment (PJB3005) [inline code/game/mecha/mecha.dm]: Use `FALSE` here.

**Review by PJB3005 — DISMISSED** [review]

**Comment by Hugh-G-Rection** [comment]
Huge balance change, will be interesting to see how this plays out. I also hate to be that guy, but this won't fit realistically on the Clarke because it uses treads for movement.

**Comment by Accipitermaris** [comment]
This should also make fighting xenomorphs in mechs far more viable because it would allow you to b-back that ass up and defend yourself with the clamp at the same time

**Comment by Ephx1** [comment]
The Clarke could just swivel like a tank.

**Pull request merged**